src/main/kotlin/org/jitsi/jibri/selenium/JibriSelenium.kt

```diff
@@ -267,7 +272,11 @@ class JibriSelenium(
             }
         }
         logger.info("Leaving web call")
-        CallPage(chromeDriver).leave()
+        try {
+            CallPage(chromeDriver).leave()
+        } catch (t: Throwable) {
+            logger.error("Error trying to leave the call", t)
+        }
         currCallUrl = null
         logger.info("Quitting chrome driver")
         chromeDriver.quit()
```

Comment:
I heard something about hung chromedrivers. Is there any force quitting logic necessary?

Reply:
this is what we get from the API.